spec/rails_helper.rb

```diff
@@ -43,25 +55,23 @@
   TEST_HOST='essi.docker'
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
     chromeOptions: {
-      args: %w[headless disable-gpu no-sandbox whitelisted-ips window-size=1400,1400]
+      args: %w[disable-gpu no-sandbox whitelisted-ips window-size=1400,1400] #run without headless
```

Comment:
can you expand this comment with the why? #run without headless so we can see screenshots (for example)